pageserver/src/tenant/span.rs

```diff
@@ -1,24 +1,20 @@
+use utils::tracing_span_assert::{check_fields_present, MultiNameExtractor};
```

Comment:
Ah... clippy nags in release mode, so need either:

```suggestion
#[cfg(debug_assertions)]
use utils::tracing_span_assert::{check_fields_present, MultiNameExtractor};
```

Or wrapping the whole thing in a:

```
cfg_if::cfg_if! {
  if #[cfg(debug_assertions)] {
    use ...
    pub(crate) static TENANT_ID_EXTRACTOR: ...;
  } else {
    pub(crate) fn debug_assert_current_span_has_tenant_id() {}
  }
}
```

Probably easier to just add more annotations, it's not like this file would have any reason to change in any future.

Reply:
Similarly in other file.

Reply:
We dont have cfg-if anywhere, so just added one more debug_assertion cfg